.changeset/dull-bobcats-beam.md

```diff
@@ -0,0 +1,9 @@
+---
+"@atproto/dev-env": patch
+"@atproto/ozone": patch
+"@atproto/api": patch
+"@atproto/pds": patch
+"@atproto/xrpc-server": patch
```

Comment:
I think ideally the change to xrpc-server would be its own separate changeset, with a description that will make sense in context of the xrpc-server changelog.